README.md

```diff
@@ -1 +1,219 @@
-# HelloMicroverse
\ No newline at end of file
+# HelloMicroverse
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Micoverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse<a name="about-project"></a>
+
+
+**Hello Microverse** is the first project of module one in the microverse schedule. It carries the git workflows, setting up linter for html and css.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### HTML and CSS <a name="html"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```

Comment:
- [ ] Kindly remove these comments since it was just there for your guidance, keeping them will make your `README` look unprofessional.